src/display/text_layer.js

```diff
@@ -668,8 +668,8 @@ var renderTextLayer = (function renderTextLayerClosure() {
             div.style.transform = transform;
           }
         } else {
-          div.style.padding = 0;
-          div.style.transform = divProps.originalTransform || '';
+          div.style.padding = null;
+          div.style.transform = divProps.originalTransform;
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style#Setting_styles:
> A style declaration is reset by setting it to `null` or an empty string, e.g., `elt.style.color = null`.

Hence the empty string isn't necessary here, since `originalTransform` defaults to `null`. Furthermore by not setting the `padding` to zero we can avoid `style =" ... padding: 0px; ..."` everywhere after expanding is reverted.